setup_go.sh

```diff
@@ -20,3 +20,7 @@ git clone --branch "go$GO_VERSION" --depth 1 https://go.googlesource.com/go goro
 
 # Update PATH to include the built go binaries
 echo 'export PATH="/goroot/bin:$PATH"' >> /root/.bashrc
+
+# Remove gimme
+rm -rf $HOME/.gimme
+rm /bin/gimme
```

Comment:
`gimme` is [used in the security agent build](https://github.com/DataDog/datadog-agent/blob/e57ba47144922e7c779d55d13083aead7edc6b94/tasks/security_agent.py#L45), but I am not sure why removing it here did not make the tests fail :thinking:

Reply:
It's used there only if we want to override the go version, something we never do in the CI and that to my knowledge no one ever does locally. I will do a PR to remove this thing if unused

Reply:
Ah, right, didn't notice the `if go_version`.

Reply:
This should be good now, the use in security agent was removed